matterhook/matterhook.go

```diff
@@ -159,7 +158,10 @@ func (c *Client) Send(msg OMessage) error {
 	defer resp.Body.Close()
 
 	// Read entire body to completion to re-use keep-alive connections.
-	io.Copy(ioutil.Discard, resp.Body)
+	_, err2 := io.Copy(io.Discard, resp.Body)
+	if err2 != nil {
+		return err2
+	}
```

Comment:
No need for a different variable name, usually you just reassign the err variable
```suggestion
	_, err = io.Copy(io.Discard, resp.Body)
	if err != nil {
		return err
	}
```

Reply:
The linter complained until I added it. Please run the linter against the file without it first before suggesting I remove it.

Reply:
I switched it back and it's not showing the warning any more so yeah you're right.